lib/libs/email/content/upload-subsequent-documents/emailTemplates/ChipSpaState.tsx

```diff
@@ -0,0 +1,44 @@
+import { Events } from "shared-types";
+import { CommonEmailVariables } from "shared-types";
+import { Text } from "@react-email/components";
+import {
+  PackageDetails,
+  ContactStateLead,
+  DetailsHeading,
+  Attachments,
+} from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
+import { styles } from "../../email-styles";
+
+export const ChipSpaStateEmail = (props: {
+  variables: Events["UploadSubsequentDocuments"] & CommonEmailVariables;
+}) => {
+  const variables = props.variables;
+  const previewText = `Additional documents submitted for CHIP SPA ${variables.id}`;
+  const heading = `You’ve successfully submitted the following to CMS reviewers for CHIP SPA ${variables.id}`;
+
+  return (
+    <BaseEmailTemplate
+      previewText={previewText}
+      heading={heading}
+      applicationEndpointUrl={variables.applicationEndpointUrl}
+      footerContent={<ContactStateLead isChip />}
+    >
+      <DetailsHeading />
+      <PackageDetails
+        details={{
+          "State or territory": variables.territory,
+          Name: variables.submitterName,
+          Email: variables.submitterEmail,
```

Comment:
Make sure it's `Email` and not `Email address`

Reply:
"Email Address" right? If so I made the correction thanks